src/sess.rs

```diff
@@ -1370,6 +1370,17 @@ pub enum DependencySource {
     Git(String),
 }
 
+impl<'a> From<&'a config::Dependency> for DependencySource {
+    fn from(cfg: &'a config::Dependency) -> DependencySource {
+        match *cfg {
+            config::Dependency::Path(ref path) => DependencySource::Path(path.clone()),
+            config::Dependency::GitRevision(ref url, _) => DependencySource::Git(url.clone()),
+            config::Dependency::GitVersion(ref url, _) => DependencySource::Git(url.clone()),
+            config::Dependency::Version(_) => DependencySource::Git("not linked".to_string()),
```

Comment:
I think this should be
```suggestion
            config::Dependency::Version(_) => DependencySource::Registry,
```
like here:
https://github.com/pulp-platform/bender/blob/22784177e742d1bfb1543969a1241f6d1bd6587c/src/sess.rs#L133-L136

I suggest to update the `impl From` with the latter code and use the `From` trait where the latter code is currently used.